src/main/java/com/airlines/dao/CrewDaoImp.java

```diff
@@ -0,0 +1,101 @@
+package com.airlines.dao;
+
+import com.airlines.entity.Citizenship;
+import com.airlines.entity.Crew;
+import com.airlines.entity.CrewMember;
+import com.airlines.entity.Position;
+import com.airlines.exception.DaoOperationException;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CrewDaoImp implements CrewDao {
+    private static final String INSERT_CREWS_CREW_MEMBERS = "INSERT INTO crews_crew_members (crews_id, crew_members_id) VALUES(?, ?);";
+    private static final String SELECT_CREW_MEMBERS_BY_CREW__ID =
+    "SELECT * FROM crew_members  " +
+            "LEFT JOIN crews_crew_members " +
+            "ON crew_members.id = crews_crew_members.crews_members_id " +
+            "WHERE crews_crew_members.crews_id=?;";
+
+    private static final String SELECT_CREW_MEMBERS_BY_CREW__NAME =
+               "SELECT *" +
```

Comment:
"SELECT * "
should be space after *

Reply:
fixed in ed36dce27a52f51b2246b22e6eb254afa9277071